src/app/components/LiveLabel/index.tsx

```diff
@@ -50,15 +51,23 @@ const LiveLabel = ({
   }
 
   const circle = [
-    [styles.liveLabelCircle, className === 'first-promo' && styles.firstPromo],
+    [styles.pulseContainer, className === 'first-promo' && styles.firstPromo],
   ];
 
   return (
     // The id below is a temporary fix for the a11y nested span's bug experienced in TalkBack, refer to the following issue: https://github.com/bbc/simorgh/issues/9652
     // eslint-disable-next-line jsx-a11y/aria-role
     <span id={id} role="text">
       <span css={circle}>
-        <span css={styles.livelabelPulse} />
+        <svg
+          fill={LIVE_DARK}
```

Comment:
Another thought RE android's dark mode (although is this officially supported?)

If we were to add the colour css to the outer span, and then have fill=currentColor inside the svg attributes, (and then remove the color from the `styles.liveLabelText` would that also make it easier to change the colour of the entire component without having to change it in multiple places? 

We could have a look to see what it might look like on Storybook / Preview?

Reply:
I had a play around with the `isDarkUi` parameter in the css so that if the page does somehow end up on dark mode then it has conditional logic to cover it (before & after screenshots below from storybook/local browser). I used a browser extention to force the page to dark mode and voilaaaa, light teal! 

**Before/current:** 

![Screenshot 2023-12-14 at 18 34 42](https://github.com/bbc/simorgh/assets/120198886/1887f0ae-4e90-492d-b9ec-9ccb983fe9c2)

**After:**

![Screenshot 2023-12-14 at 18 35 09](https://github.com/bbc/simorgh/assets/120198886/0606f1e5-75fc-4aca-a311-e1d01ca5b3e5)

![Screenshot 2023-12-14 at 18 43 15](https://github.com/bbc/simorgh/assets/120198886/5a90d8f1-af3a-46a4-9803-b52ee7c47619)


However it still works even when I removed the logic when testing, so it seems that now that the pulse and live label text are wrapped together in a span then it just automatically knows to change somehow? 🤔